### DISCUSSION TITLE
IOS does not import individual tracks from .gpx file #1287

### DISCUSSION THREAD

**Discussion by drtbyk** [osmandapp/OsmAnd#13500] [Maps Related]
When importing a GPX file, with multiple GPX Tracks, into OsmAnd-iOS, the App imports all of the individual Tracks as a single Track entry in "My Places - Tracks". To make things even worse, the App removes the individual Track NAMES and instead calls them "Segments(n+1)" under the GPX file Name.

This is not a problem on OsmAnd-Android - this app imports the same GPX file used above as multiple separate Tracks with the proper NAMES. 

This issue has been know for as long as OsmAnd-iOS has been available. When I recommend that someone take a look at OsmAnd-iOS, the first thing they tell me is that it doesn't know how to import Tracks properly - and they never come back.

It's really a shame that this issue has had such an negative impact on the use of OsmAnd-iOS.

If you would like to see this issue in action, you can use the attached file.

[Tracks_-_Multiple_Separate_ 2.zip](https://github.com/osmandapp/OsmAnd/files/7833811/Tracks_-_Multiple_Separate_.2.zip)

Cheers,

**Comment by vshcherb** [comment]
Yes, we're planning to implement it as well. Not present yet in Android and in iOS
